src/views/Home.jsx

```diff
@@ -13,13 +47,26 @@ export function Home({ data, setListPath }) {
 					return (
 						<SingleList
 							key={list.name}
-							name={list.path}
-							path={list.name + '/' + list.path}
+							name={list.name}
+							path={list.path}
 							setListPath={setListPath}
 						/>
 					);
 				})}
 			</ul>
+			<form>
```

Comment:
Thanks for keeping this form simple! Remember, the team will be adding styles, etc later down the road.